runtime/doc/provider.txt

```diff
@@ -135,6 +135,7 @@ currently. The Vim legacy |perl-vim| interface is also supported (which is
 itself implemented as a Nvim remote-plugin).
 https://github.com/jacquesg/p5-Neovim-Ext
 
+Note: Only perl versions from 5.22 onward are supported.
```

Comment:
Make this is an explicit requirement and update the health check.

Reply:
Done.